tests/foreman/ui/test_repository.py

```diff
@@ -65,14 +67,23 @@ class RepositoryTestCase(UITestCase):
     @classmethod
     def setUpClass(cls):  # noqa
         super(RepositoryTestCase, cls).setUpClass()
-        cls.organization = entities.Organization().create()
-        cls.loc = entities.Location().create()
+        # create instances to be shared across the sessions
+        cls.session_loc = entities.Location().create()
+        cls.session_prod = entities.Product(
+            organization=cls.session_org).create()
+
+    @classmethod
+    def set_session_org(cls):
```

Comment:
@oshtaier Yes, this is called in `setupClass` specially to make the user to be logged in automatically with that org context.